client/components/planning-editor-standalone/profile-fields.ts

```diff
@@ -21,6 +21,11 @@ const unimplementedFields = new Set<string>([
     'associated_event',
 ]);
 
+const systemRequiredFields = new Set<string>([
```

Comment:
it already displays in the content profile which fields are system fields. You should use the same data source here as well instead of duplicating